src/specs.ts

```diff
@@ -50,9 +43,46 @@ export async function specsFromOpts(opts: Opts): Promise<BenchmarkSpec[]> {
     }
   }
 
-  const versions = parsePackageVersions(opts['package-version']);
+  const remoteUrls = [];
+  const localNames = [];
+  // Benchmark names/URLs are the bare arguments not associated with a flag, so
+  // they are found in _unknown.
+  for (const benchmark of opts._unknown || []) {
+    try {
+      new url.URL(benchmark);
+      remoteUrls.push(benchmark);
+    } catch (e) {
+      if (e.code === 'ERR_INVALID_URL') {
+        localNames.push(benchmark);
+      } else {
+        throw e;
+      }
+    }
+  }
 
   const specs: BenchmarkSpec[] = [];
+  for (const url of remoteUrls) {
+    for (const browser of browsers) {
+      specs.push({
+        url,
+        measurement: 'fcp',
+        browser,
+        // TODO Find a shorter unambiguous name since these can make the result
+        // table unwieldy, or do something smarter in the result table.
```

Comment:
Maybe a key=value pair in the arguments?

Reply:
Yeah, to assign an arbitrary label to each URL? Sounds right to me.